libs/SalesforceReact/package.json

```diff
@@ -0,0 +1,20 @@
+{
```

Comment:
NB: package.json is under libs/SalesforceReact
We don't want to be running the react native packager at the root of the repo otherwise it would find (and be confused by) the javascript code for cordova.